lib/watchman.rb

```diff
@@ -12,8 +12,8 @@ class << self
     attr_accessor :port
     attr_accessor :test_mode
 
-    def submit(name, value, type = :gauge)
-      metric = metric_name_with_prefix(name)
+    def submit(name, value, type = :gauge, tags_list = [])
```

Comment:
As far as I know, we use it only in one place. A masterful `sed -i` would solve everything ;)

We should never be afraid to write good software :D <3